### PR TITLE
Frontend testable code

### PR DESCRIPTION
**Pull request by Kris2ffer**
Functionality has been moved from study page and pod page to it's own file where it is easy to test.

### CODE REVIEW

File: FrontEnd/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "start": "set PORT=8080 && react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test:coverege": "react-scripts test --coverage",
```
Comment (gjersvik) [inline FrontEnd/package.json]: need to add --watchAll=false for coverage to work.
Also typo in test:coverege.

File: FrontEnd/package.json [hunk 2]
```diff
@@ -29,5 +30,15 @@
...
     ]
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/*.{js,jsx}",
+      "!**/node_modules/**",
+      "!**/coverage/**",
+      "!**/serviceWorker.js",
+      "!**/index.js",
+      "!**/App.js"
```
Comment (gjersvik) [inline FrontEnd/package.json]: Have mixed feelings if these files should be ignored here or by ignore comment.
```
/* istanbul ignore file */ to ignore file.
/* istanbul ignore next */ to ignore the next thing. Function, line, class.
```

File: FrontEnd/src/__tests__/App.test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  // setup a DOM element as a render target
+  div = document.createElement("div");
+  document.body.appendChild(div);
+});
+
+
+it('renders without crashing', () => {
+  act(() => {
+    ReactDOM.render(<App />, div);
+  })
```
Comment (gjersvik) [inline FrontEnd/src/__tests__/App.test.js]: This test results in errors even if it passes that send mixed messages. I would ignore it until you can create is safely.

**Review by gjersvik — APPROVED** [review]
Look good have just added some pointers and thoughts for future work.
One general ting. files that contains react <div ... > syntax should be called .jsx as they are not valid .js files.

**Review by Unicus-Tom — APPROVED** [review]

**Pull request merged**